pkg/pgmodel/model/interface.go

```diff
@@ -49,7 +49,7 @@ func MetricTableName(conn pgxconn.PgxConn, metric string) (string, bool, error)
 	var possiblyNew bool
 	defer res.Close()
 	if !res.Next() {
-		if res.Err() != nil {
+		if err := res.Err(); err != nil {
 			return "", true, err
```

Comment:
note this is where the old code could have returned a nil error with a blank tableName